src/anal_ghidra.cpp

```diff
@@ -381,6 +381,7 @@ static ut32 anal_type_XCMP(RAnal *anal, RAnalOp *anal_op, const std::vector<Pcod
 	uintb unique_off = 0;
 	PcodeOpType key_pcode = CPUI_MAX;
 
+	anal_op->type = R_ANAL_OP_TYPE_CMP;
```

Comment:
Why set `R_ANAL_OP_TYPE_CMP` here? Could you provide binary sample?

Reply:
Because all the instructions processed  by this functions are always CMPs, and disassembling v850 cmps never falls in the case where this field is set. So im just saving the default value. There are other instructions that have the same issue. Ill update the PR accordingly

Reply:
and nope. i cant provide any binary, but i can provide a random sequence of bytes to repro

Reply:
Yes, please add a test to `test/db/extras/anal_ghidra` for this.